### PR TITLE
Feature/343 NSHM_V1.0.4 model version update

### LINKED ISSUE #343: Move Kororaa to NSHM v1.0.4
We want to serve version 1.0.4 of the NSHM to users:

- [ ] remove the "Bug Fix Notice" and it's following text from the disclaimer
- [ ] Footer should read that user is viewing `NSHM_v1.0.4`
- [ ] All hazard data (Curves and Spectra, Hazard Maps) should point to model ID `NSHM_v1.0.4`
- [ ] Disaggregations should be for `NSHM_v1.0.4` (@chrisdicaprio)
- [ ] Change location of Franz Josef in disaggs to srg_164
- [ ] New entry in **/ModelVersions** (text below)
- [ ] Change location of Franz Josef for Curves and Spectra (if looking up by id it is `srg_164`. Coordinates are  'latitude': -43.375755162, 'longitude': 170.18836548)
- [ ] Change entry for Franz Josef in the location table for **/TechInfo** (use -43.38, 170.19)

**/ModelVersions** Text:

### Fixed
- Fault dip direction for crustal faults
- Dip orientation of Puysegur fault sections corrected.
- Kermadec interface location corrected. A bug had placed sources east of 180 degrees longitude in the wrong position.
- Statistical aggregation of hazard curves. A bug introduced a small error (<1%) in the mean hazard curve at high probability / low values of shaking

## NSHM_v1.0.0 - 2022-10-04

Initial release of the NZ NSHM 2022 revision

### Bug Advisory: April 2023

We have identified some errors in NSHM_v1.0.0. These errors are being corrected and are unlikely to affect any NSHM results-based decision making.

A bug in internationally sourced computer modelling code affected fault dips in the final step of the hazard calculations and affected every result from the modelling. However, tests show that the correction will introduce negligible hazard changes for most of the country. One particular section of the Alpine Fault was affected, and the fault was moved farther or nearer to certain locations. The mean hazard estimate for Franz Joseph has effectively remain unchanged while the mean hazard estimate for Aoraki/Mt Cook has increased. These changes are unlikely to affect any NSHM results-based decision making.

### PR DESCRIPTION
**Pull request by benjamineac**
closes #343

### CODE REVIEW

**Review by chrisbc — COMMENTED** [review]
Scanned - looks OK

File: src/views/info/ModelVersionsPage.tsx [hunk 1]
```diff
@@ -24,17 +24,23 @@ const TitleContainer = styled('div')({
 });
 
 const markdown = `
+## NSHM_v1.0.4 - 2023-05-05
```
Comment (chrisbc) [inline src/views/info/ModelVersionsPage.tsx:27]: TODO: split up versions as per @chrisdicaprio comment on main ticket

File: src/utils/environmentVariables.ts [hunk 2]
```diff
@@ -62,4 +62,4 @@ export const LOWER2: string = process.env.REACT_APP_LOWER2 || '0.05';
 export const GA_ID: string | undefined = process.env.REACT_APP_GA_ID;
 export const GA_DEBUG_MODE: boolean = process.env.REACT_APP_GA_DEBUG_MODE === 'true';
 
-export const HAZARD_MODEL_VERSION = 'NSHM_v1.0.0';
+export const HAZARD_MODEL_VERSION = process.env.REACT_APP_HAZARD_MODEL || 'NSHM_v1.0.0';
```
Comment (chrisbc) [inline src/utils/environmentVariables.ts:65]: #TODO remove this and update all affected components (use HAZARD_MODEL instead)

File: src/views/techinfo/TechInfoPage.tsx [hunk 3]
```diff
@@ -55,7 +55,7 @@ const rows = [
   createData('Taupo', -38.68, 176.08),
   createData('Rotorua', -38.14, 176.25),
   createData('Whakatane', -37.98, 177.0),
-  createData('Franz Josef', -43.35, 170.17),
+  createData('Franz Josef', -43.376, 170.188),
```
Comment (chrisbc) [inline src/views/techinfo/TechInfoPage.tsx:58]: #todo alpha sort this list

**Review by chrisbc — COMMENTED** [review]
scanned OK.

**Review by chrisdicaprio — APPROVED** [review]

**Pull request merged**